Minecraft/Bot.ts

```diff
@@ -1,9 +1,9 @@
-import { createClient } from '../Utils/DrippyBlockClient';
-import { DrippyClient } from '../Utils/DrippyClient';
+import { createClient } from '../Utils/MinecraftClient';
+import { DiscordClient } from '../Utils/DiscordClient';
 import { HandleChat } from './Chat';
 import { PlayerManager } from './PlayerManager';
 
-export function HandleMinecraft(client: DrippyClient) {
+export function HandleMinecraft(client: DiscordClient) {
```

Comment:
I might've missed some other places were this is the case but please use lowerCamelCase for functions (start them with a lower case letter

Reply:
K